superset-frontend/src/explore/controlPanels/Bubble.js

```diff
@@ -39,15 +72,22 @@ export default {
       expanded: true,
       controlSetRows: [
         ['color_scheme', 'label_colors'],
-        ['show_legend', null],
+        [showLegend, null],
       ],
     },
     {
       label: t('X Axis'),
       expanded: true,
       controlSetRows: [
-        ['x_axis_label', 'left_margin'],
-        ['x_axis_format', 'x_ticks_layout'],
+        [xAxisLabel, leftMargin],
+        [
+          {
+            ...xAxisFormat,
```

Comment:
`xAxisFormat` is `{name, config}`.

Perhaps this should be changed to
```ts
{
  name: xAxisFormat.name,
  config: {
    ...xAxisFormat.config, 
    default: 'SMART_NUMBER',
    choices: D3_FORMAT_OPTIONS,
  }
}
```

Reply:
Well, now, THAT is... just an old fashioned mistake. 🙈